docs/docs/workflows/improve-your-data/remove-duplicate-images.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 4
+---
+
+# Removing Duplicate Images
+
+**Improve your dataset by detecting and removing duplicate and near-duplicate images**
+
+Duplicate or near-duplicate images may cause bias in deep learning models. Encord-Active enables you to find and remove
+duplicate and near duplicate images in your dataset so that you can remove them and improve the overall quality of the 
+data. 
+
+`Prerequisites:` Dataset
+
+## Image singularity
+
+This metric checks all your images in the dataset and assigns a score to them, which shows
+their uniqueness.
+
+- Uniqueness score is in the range of [0,1]. A higher value means a more unique image.
+- A zero value indicates that this image has at least one identical image in this dataset. In the case of _N_ 
+duplicate images, we assign a value of zero to _N-1_ of them (only one of them will have a nonzero value) so that they
+can be excluded from the dataset immediately.
+- In the case of duplicate images, _N-1_ image will be linked to a single image among those and this image can be 
+visualized using the `Description` field under each image.
+- Near duplicate images are labeled as `Near duplicate image` and shown side by side in the grid view. So that
+users can choose which one to keep and which to remove.
+
+## Steps
+Navigate to the _Data Quality_ > _Explorer_  tab and select the **Image Singularity** quality metric in the top left 
+combobox to order your data. 
+
+![remove-duplicateimages-1](../../images/workflows/improve-your-data-and-labels/remove-duplicate-images/remove-duplicate-images-1.png)
+
+In the dashboard, you can see the distribution of your data according to the image singularity scores.
+
+There are approximately 200 duplicate images in this example.
+
+![remove-duplicateimages-2](../../images/workflows/improve-your-data-and-labels/remove-duplicate-images/remove-duplicate-images-2.png)
+
+Go a little bit down and see the duplicate images. If you click **show similar images** button, you should see at least
+one image in the similar ones. If you have [Encord Annotate](https://encord.com/) account, you can directly inspect the other image via
```

Comment:
I think that this is more a matter of whether the project is associated to Encord Annotate right?

Reply:
My point was doing something like this:
```suggestion
one image in the similar ones. If you have a project which is imported from [Encord Annotate](https://encord.com/), you can directly inspect the other image via
```